airlytix-es1.yaml

```diff
@@ -631,7 +631,7 @@ number:
     name: CO2 Good Min
     icon: mdi:arrow-left-right
     entity_category: config
-    min_value: 400
+    min_value: 0
     max_value: 5000
     initial_value: 400
```

Comment:
Can you set the `initial_value` for good min to 0 as well so future installs won't need to lower the value

Reply:
done